installer_scripts/install-yb-voyager

```diff
@@ -299,6 +304,1280 @@ update_bashrc() {
 	done
 }
 
+insert_yb_session_vars_file() {
+	sudo touch /etc/ybSessionVariables.sql
+	sudo sh -c 'printf "SET client_encoding TO '\''UTF8'\'';\nSET session_replication_role TO replica;" > /etc/ybSessionVariables.sql'
+}
+
+insert_ora2pg_conf() {
+conf_file_name="${HOME}/.yb-voyager-base-ora2pg.conf"
```

Comment:
To copy the embeded ora2pg.conf file you can try these command from shell script:
```
repoAbsPath=$(git rev-parse --show-toplevel)
cp $repoAbsPath/yb-voyager/src/srcdb/data/sample-ora2pg.conf /etc/path/to/destination/
```

Reply:
Will try this, thanks for letting me know

Reply:
Please review the rest of the PR, I believe we discussed later on as to why this approach may not be feasible either